ignite/engine/engine.py

```diff
@@ -433,7 +433,7 @@ def terminate_epoch(self) -> None:
         )
         self.should_terminate_single_epoch = True
 
-    def _run_once_on_dataset(self) -> Tuple[int, int, int]:
+    def _run_once_on_dataset(self) -> float:
```

Comment:
@erip could you please revert this

Reply:
If this is reverted, we'll need to unparse the tuple to a timestamp unless we want to put the state-modifying code in the `_run_once_on_dataset`.

Reply:
Yes, you are right ! I didn't get it completely, sorry.

Reply:
Oh, so I should keep it as is with `-> float`? Since it's an internal method it shouldn't break anything (or anyone).